storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -4896,8 +4896,21 @@ static void xtrabackup_stats_func(int argc, char **argv) {
     exit(EXIT_FAILURE);
   }
 
+  /*
+  auto log_files_8_30_pre_ctx =
+      Log_files_context{srv_log_group_home_dir, Log_files_ruleset::PRE_8_0_30};
+  ut::vector<Log_file_id> listed_files_8_30;
+  if (log_list_existing_files(log_files_8_30_pre_ctx, listed_files_8_30) ==
+      DB_SUCCESS) {
+    xb::error() << "This version of Xtrabackup Cannnot not take statsistics  "
+                   "for version below 8.0.30";
+    exit(EXIT_FAILURE);
+  }
+  */
```

Comment:
isn't it better if we check for the number of files?

Reply:
Hmm didn't realize that this code is commented :scream:  shouldn't this be enabled?

Reply:
Fixed.